script/clippy.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
Can we move this clippy functionality to maybe `build.sh`? Because now there's some duplicated values and functionality here, and so ideally we don't have that.

Reply:
It is now part of the `build.sh` script